test/HarshPoint.Shellploy.Generator.Tests/CommandBuilding/Without_configuration.cs

```diff
@@ -62,6 +61,50 @@ public void Has_no_Namespace()
             Assert.Null(_command.Namespace);
         }
 
+        [Fact]
+        public void Has_CmdletAttribute()
+        {
+            Assert.Single(
+                _command.Attributes
+                    .Where(attr =>
+                        attr.AttributeType == typeof(SMA.CmdletAttribute))
+            );
+        }
+
+        [Fact]
+        public void Has_OutputTypeAttribute()
```

Comment:
to jsem udelala, aby to bylo rozepsane do vice testu; do tech ostatnich jsem Single pridala az pozdeji